apps/storybook/src/gp2-components/UserCard.stories.tsx

```diff
@@ -32,24 +33,29 @@ export const Normal = () => {
     'Africa',
   );
   return (
-    <UserCard
-      id="u42"
-      displayName={text('Display Name', 'Phillip Mars')}
-      degree={degree}
-      firstName={text('First Name', 'Phillip')}
-      lastName={text('Last Name', 'Mars')}
-      avatarUrl={text('Avatar URL', '')}
-      role={'GP2 Admin'}
-      region={region}
-      workingGroups={Array(number('Number of Working Groups', 1)).fill({
-        id: 't42',
-        name: text('Working Group Name', 'Underrepresented Populations'),
-      })}
-      projects={Array(number('Number of Projects', 1)).fill({
-        id: 't42',
-        name: text('Project Name', 'Genetic determinants of progression in PD'),
-      })}
-      tags={tags}
-    />
+    <Theme>
```

Comment:
We should think about making this component reusable in storybook so its easy to switch themes on any component that needs to